elizhahn/karaoke-stretch
# Styling/sass architecture

## What does this PR do (summary of changes)?

 - Applies gradient and font-family to App Component and specific children where necessary
 
## How should it be tested?

 - User can observe dynamic changes to the nav bar and cards as they navigate the application

## Future Iterations:

 - Modify for responsiveness
